src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj

```diff
@@ -56,7 +56,7 @@
     <Reference Include="System.Net.Http" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Client.NativeInterop" Version="0.12.3" />
+    <PackageReference Include="Microsoft.Identity.Client.NativeInterop" Version="0.12.4" IncludeAssets="all" />
```

Comment:
> Also include runtime, build and lib assets by adding all.

Should only the specific ones be included then? `IncludeAssets=""build;runtime;compile`

Reply:
We only have bin, lib and runtime. Will all make sure these three are included? or should we be specific?

Reply:
https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets

I think all should work. I wasn't sure if all would include extra things we don't need, so was trying to clarify with you. :)